### PR TITLE
fix(learn): changed test to be different from the video

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

This PR was created for the sole purpose of implementing the changes of PR #39747 which needed to be rebased.  Since user created branch on `master`, I did not want to force push back to it.

@raisedadead This PR should be good to go once all the tests pass.

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

**Comment by RandellDawson** [comment]
@Sky020 If you wouldn't mind validating this PR does contain the same changes as PR #39747, that would be great.  If so, go ahead merge.  Thanks.

File: curriculum/challenges/english/08-data-analysis-with-python/numpy/initialize-array-problem.md [hunk 1]
```diff
@@ -20,42 +20,44 @@ question:
...
+      output[1:1, -1:-1] = z
       ```
     - |
       ```py
-      output = np.ones((5, 5))
+      output = np.zeros((7,7))
 
-      z = np.zeros((3, 3))
-      z[1, 1] = 9
+      z = np.zeros((5, 5))
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/numpy/initialize-array-problem.md:46]:
```suggestion
      z = np.ones((5, 5))
```

**Review by ShaunSHamilton — APPROVED** [review]
LGTM now 👍

**Comment by RandellDawson** [comment]
@Sky020 Thanks for fixing that.  I just realized that it was never fixed in the original PR this one replaces.  It has been a long day.

**Comment by Sky020** [comment]
@RandellDawson  No worries 😃 I am just waiting on the tests, before merging. Thanks for porting this over.

**Comment by Sky020** [comment]
Congrats, @nitnjain  on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**